src/components/swimlanes/swimLanes.tsx

```diff
@@ -29,10 +29,13 @@ const SwimLanes = ({ lanesWithNumItems }) => {
           type="standalone"
           href={`${DC_URL}/collections/lane/${lane.slug}`}
           aria-label={`See more ${lane.title.toLowerCase()}`}
+          hasVisitedState={false}
           sx={{
             display: { sm: "none", md: "inline" },
-            fontWeight: "500",
+            color: "ui.blue",
```

Comment:
This should be `ui.primary.link`.

The bigger concern is that the styles for this "See more" link should be the same as the other "See more" link. This link is the desktop one since it is hidden on mobile (display: "none"). Use the same pattern as the other link and use `__css` to have the same styles and pattern.